doc/api/deprecations.md

```diff
@@ -950,6 +950,13 @@ deprecated if the assigned value is not a string, boolean, or number. In the
 future, such assignment may result in a thrown error. Please convert the
 property to a string before assigning it to `process.env`.
 
+<a id="DEP0105"></a>
+### DEP0105: tls.convertNPNProtocols()
+
+Type: Runtime
+
+This was never a documented feature.
```

Comment:
Nit: It would be great if you could add a sentence explaining why we decided to remove the API instead of documenting it.

Reply:
Agree. NPN is a documented feature making this comment a bit confusing.